LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Joe Cross
+Copyright (c) 2016 Joe Cross
```

Comment:
I suspect you prpobably [don't need to update this](http://stackoverflow.com/a/2391555/400691).